gooddata-pandas/gooddata_pandas/result_convertor.py

```diff
@@ -197,14 +197,19 @@ def _read_complete_execution_result(
 
 
 def _create_header_mapper(
-    response: BareExecutionResponse, dim: int, label_overrides: Optional[LabelOverrides] = None
+    response: BareExecutionResponse,
+    dim: int,
+    label_overrides: Optional[LabelOverrides] = None,
+    use_local_ids_in_headers: Optional[bool] = False,
```

Comment:
One last thing, you can remove Optional. Optional says that the value can be None and I think that we do not want that in this case.